### PR TITLE
Chore/ block history

### LINKED ISSUE #847: Block Distribution and Missed blocks not working on Libre
This page is not loading any results

### PR DESCRIPTION
**Pull request by Torresmorah**
### What does this PR do?

- Clear blocks older than 90 days from libre testnet block history
- Resolve #847

### Steps to test

1. Run the project locally
2. Check the table block_history
3. Go to the block distribution page
4. Check the graphic

### Screenshot

imagen

### CODE REVIEW

File: .github/workflows/deploy-libre-testnet.yaml [hunk 1]
```diff
@@ -83,6 +83,8 @@ jobs:
           HAPI_EOS_API_NETWORK_NAME: libre
           HAPI_EOS_API_ENDPOINTS: '["https://libre-testnet.edenia.cloud","https://api.testnet.libre.cryptobloks.io","https://libre-testnet.eosphere.io"]'
           HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT: ''
```
Comment (xavier506) [inline .github/workflows/deploy-libre-testnet.yaml:85]: do we need a value for the SHIP endpoint?

Comment (xavier506) [reply .github/workflows/deploy-libre-testnet.yaml:85]: @Torresmorah  please add `ws://api-node.libre-testnet:8080` as value for `HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT`

**Review by leisterfrancisco — APPROVED** [review]

File: hapi/src/services/state-history-plugin.service.js [hunk 2]
```diff
@@ -149,38 +149,6 @@ const cleanOldBlocks = async () => {
   await hasuraUtil.request(mutation, { date })
 }
 
-const worker = async () => {
```
Comment (xavier506) [inline hapi/src/services/state-history-plugin.service.js]: @Torresmorah  are you sure this wasnt being used for missed blocks feature?

Comment (Torresmorah) [reply hapi/src/services/state-history-plugin.service.js]: Yes, I had added it to get the block via get_block, but the plugin does that function

**Pull request merged**